Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -1080,8 +1080,9 @@ void HackStudioWidget::build()
         GUI::MessageBox::show(window(), String::formatted("{}", result.error()), "Build failed", GUI::MessageBox::Type::Error);
         m_build_action->set_enabled(true);
         m_stop_action->set_enabled(false);
-    } else
+    } else {
         m_stop_action->set_enabled(true);
+    }
```

Comment:
As this commit is a correction of the previous one, please squash them together. [This](https://www.youtube.com/watch?v=ElRzTuYln0M) is a good guide if you don't know how to do that. Or you can ask in #development on Discord.